EventGenerator/src/GeneratorPlots_module.cc

```diff
@@ -81,12 +81,10 @@ namespace mu2e {
 
     if(!findData(event)) 
       throw cet::exception("RECO")<<"No data in  event"<< endl; 
-
-    GetGenPartInfo(event);
+    else GetGenPartInfo(event);
 }
```

Comment:
I have a strong style preference for always using {}.  I have seen too many cases where future maintenance failed due to not paying attention to the implied {}.